File: PJDesignBackStageClient/src/app/features/portfolio/feature-shared/models/create-or-update-portfolio.ts
import { EditRequestBase } from "src/app/shared/models/bases";

export class CreateOrUpdatePortfolioRequest extends EditRequestBase {
  id?: number | null;
  title: string;
  isEnabled: boolean;
  date: Date;
  categoryIDs: number[] | null | undefined;
  photos?: string[]
  thumbnailUrl: string;
}

File: PJDesignBackStageAPI/App.Model/Portfolio/CreateOrUpdatePortfolio.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace App.Model
{
    public class CreateOrUpdatePortfolioRequest : EditRequestBase
    {
        public int? Id { get; set; }
        public bool IsEnabled { get; set; }
        public string Title { get; set; } = null!;
        public DateTime? Date { get; set; }
        public IEnumerable<int>? CategoryIDs { get; set; }
        public IEnumerable<string>? Photos { get; set; }
    }
}
